Artifacts/windows-wireshark/artifactfile.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Wireshark",
+  "description": "Installs wireshark - network trace utility",
+  "tags": [
+    "Wireshark",
+    "PCAP",
+    "Trace"
+  ],
+    "iconUri": "https://upload.wikimedia.org/wikipedia/commons/d/db/Wireshark_Icon.png",
```

Comment:
I can't leave a comment on the icon committed to this folder due to limitations in the github UI, but I wanted to point out that the icon you have copied there for Wireshark is for TeamViewer. You aren't referring to it here, so maybe you just want to remove that icon.